src/GraphQL/loaders/parseClassQueries.js

```diff
@@ -67,7 +69,8 @@ const load = function(
   }
 
   if (isFindEnabled) {
-    const findGraphQLQueryName = `find${className}`;
+    const findGraphQLQueryName =
+      className.charAt(0).toLowerCase() + className.slice(1) + 's';
```

Comment:
What do you think about using some library like https://github.com/blakeembrey/pluralize ?

Reply:
done